src/main/java/exceptions/commands/CommandWrongFormatException.java

```diff
@@ -5,11 +5,11 @@
 /**
  * Represents the exception thrown when the command does not follow the proper format.
  */
-public class WrongFormatException extends GitException {
+public class CommandWrongFormatException extends GitException {
     /**
-     * Constructs WrongFormatException.
+     * Constructs CommandWrongFormatException.
      */
-    public WrongFormatException(String command) {
+    public CommandWrongFormatException(String command) {
```

Comment:
The renaming of exceptions (e.g., WrongFormatException to CommandWrongFormatException, IncompleteCommandException to IncompleteParameterException) improves clarity regarding the exceptions' contexts. This makes it easier for developers to understand what each exception represents without delving into their implementations. Well done!